src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/DatabaseErrorPageOptions.cs

```diff
@@ -12,6 +14,28 @@ namespace Microsoft.AspNetCore.Builder
     public class DatabaseErrorPageOptions
     {
         /// <summary>
+        /// Constructs a new DatabaseErrorPageOptions.
+        /// </summary>
+        public DatabaseErrorPageOptions() { }
+
+        /// <summary>
+        /// Constructs and configures a new DatabaseErrorPageOptions.
+        /// </summary>
+        /// <param name="configureOptions">The configuration actions that will be applied.</param>
+        public DatabaseErrorPageOptions(IEnumerable<IConfigureOptions<DatabaseErrorPageOptions>> configureOptions) : this()
```

Comment:
I'm failing to understand the new pattern. Do we need to register the custom Options classes on the DI container as a singleton? Should other option types like ElmOptions have a similar constructor like this one?